pybamm/solvers/solution.py

```diff
@@ -230,3 +202,93 @@ def save_data(self, filename):
         with open(filename, "wb") as f:
             pickle.dump(self.data, f, pickle.HIGHEST_PROTOCOL)
 
+
+class Solution(_BaseSolution):
+    """
+    Class extending the base solution, with additional functionality for concatenating
+    different solutions together
+
+    **Extends**: :class:`_BaseSolution`
+
+    """
+
+    def __init__(
+        self, t, y, t_event=None, y_event=None, termination="final time",
+    ):
+        super().__init__(t, y, t_event, y_event, termination)
+
+    @property
+    def sub_solutions(self):
+        "Model used for solution"
```

Comment:
not sure this docstring is very clear?

Reply:
oops yeah that is totally a copy paste error